lib/executor/constant_arrival_rate.go

```diff
@@ -253,33 +253,42 @@ func (car ConstantArrivalRate) Run(ctx context.Context, out chan<- stats.SampleC
 	go trackProgress(ctx, maxDurationCtx, regDurationCtx, car, progresFn)
 
 	regDurationDone := regDurationCtx.Done()
-	runIterationBasic := getIterationRunner(car.executionState, car.logger, out)
-	runIteration := func(vu lib.VU) {
+	runIterationBasic := getIterationRunner(car.executionState, car.logger)
+	runIteration := func(initVU lib.InitializedVU, modActive bool) {
+		ctx, cancel := context.WithCancel(maxDurationCtx)
+		defer cancel()
+
+		vu := initVU.Activate(&lib.VUActivationParams{RunContext: ctx})
```

Comment:
This is wrong - VU activation shouldn't happen before every iteration. Activation means "the VU is prepared by the executor to run code", and should happen only once per VU per executor.

Reply:
Should be resolved by f02a56c5.